src/apps/api/tests/test_queue_api_permissions.py

```diff
@@ -104,3 +99,42 @@ def test_organizer_user_cannot_delete_or_edit_queue(self):
         )
         assert response.status_code != 204
         assert Queue.objects.filter(pk=self.created_queue.id).exists()
+
+    # Test user queue limits
+    def test_queue_limits_for_regular_users(self):
+        [QueueFactory(owner=self.user) for _ in range(3)]
+
+        self.client.login(username='test', password='test')
+        data = self._get_test_update_data(return_id=False)
+        # default limit is 5
+        response = self.client.post(
+            reverse('queues-list'),
+            data=data,
+            content_type='application/json'
+        )
+        assert response.status_code == 201
+        # Next one should fail!
+        response = self.client.post(
+            reverse('queues-list'),
+            data=data,
+            content_type='application/json'
+        )
+        assert response.status_code == 400
+
+    # Test user queue limits
+    def test_queue_limits_for_super_users(self):
+        # Make test user a superuser before hand
+        self.user.is_superuser = True
+        self.user.save()
+
+        [QueueFactory(owner=self.user) for _ in range(5)]
```

Comment:
given you create a queue in setup already, this should be `range(4)` so that the total is at the limit? or, as I said above, could probably explicitly set the user queue limit lower, so we don't have to run the factory as many times. Will also speed these test up (minimally)

Reply:
Set queue limit lower, so this is more explicit, and assert we're already at the limit before making another successful request
✔️